### DISCUSSION TITLE
How to delete a "Setup Sync"?

### DISCUSSION THREAD

**Discussion by kud** [Kong/insomnia#3900] [Help]
All is in the title. I misclicked by error.

**Comment by dimitropoulos** [comment]
@kud could you please provide some reproduction steps?   If you're talking about what I think you're talking about - see if you are able to delete it after you create (but not push) a snapshot.

**Reply by kud** [reply]
https://user-images.githubusercontent.com/655838/129692517-c0802fba-9417-4845-8807-5536af991a57.mp4

**Reply by kud** [reply]
Here you can see we can't delete the sync anymore.

**Reply by dimitropoulos** [reply]
Thank you for the video, it makes it much easier.  What you're showing is a known behavior.  I think there's some improvement to be had there (namely, that the delete should immediately be not disabled) but I'm trying to first verify that what you're asking is a feature request (for the button to not be disabled) and not a bug (that you can't delete the collection via the current path).  That "current path" is to first create a snapshot (it's in the same dropdown you showed above as "Create Snapshot") and _then_ delete.  Could you please verify for me that you are in fact able to delete a setup sync.

Again, I totally understand how this is confusing that you can't immediately delete and we can definitely improve this, but first I want to verify you're not hitting some bug or edge case we're not already aware of.

**Reply by kud** [reply]
I understand yes, thank you for your time.

My feature request yeah is to simply remove the sync and come back to the state "setup sync". Because to be frank I don't know exactly what it does to set sync up :D Probably a git repo?

My concern is that I clicked on "setup sync" on a private company project and I don't want it on the cloud (on a server I don't know). :)

**Reply by develohpanda** [reply]
Thanks for that - the way sync currently works is a local project is first created (which is what you did), and once you create a snapshot and push it ends up in Insomnia Sync (which is fully [encrypted](https://support.insomnia.rest/article/159-end-to-end-encryption)).

As long as you didn't push any snapshots, none of your data will have been synced to the cloud. 

Ideally we should allow you to disable sync while in this interim state, so this would be a bug

**Reply by kud** [reply]
Lovely, thank your this explanation @develohpanda :)
